thirdparty/github.go

```diff
@@ -271,16 +296,40 @@ func GetGithubCommits(ctx context.Context, oauthToken, owner, repo, ref string,
 	commits, resp, err := client.Repositories.ListCommits(ctx, owner, repo, &options)
 	if resp != nil {
 		defer resp.Body.Close()
+		if err == nil {
+			return commits, resp.NextPage, nil
+		}
+	}
+
+	// fallback to not using github app on error
+	grip.Debug(message.Fields{
+		"ticket":  "EVG-19966",
+		"message": "failed to get commits from GitHub",
```

Comment:
Seems like this debug message should be different and say something about falling back?